pumpkin-world/src/item/item_registry.rs

```diff
@@ -77,6 +79,12 @@ pub struct Modifier {
     pub slot: String,
 }
 
+#[derive(Deserialize, Clone, Copy, Debug)]
+pub struct Food {
+    pub nutrition: u8,
```

Comment:
Looking at Vanilla this struct should be
```Food
nutrition: u32,
saturation: f32,
can_always_eat: Optional<bool>,

Reply:
Oh, yes, i fix it when i work on pc